src/api/variables-api.ts

```diff
@@ -31,7 +31,7 @@ class VariablesApi {
     }
 
     public getRuntimeVariableValues(packageKey: string): Promise<VariablesAssignments[]> {
-        return httpClientV2.get(`/package-manager/api/nodes/by-package-key/${packageKey}/variables/runtime-values`).catch(e => {
+        return httpClientV2.get(`/package-manager/api/nodes/by-package-key/${packageKey}/variables/runtime-values?appMode=VIEWER`).catch(e => {
```

Comment:
I think this is very hidden here, and fetching `VIEWER` variable is a logic specific to the config export command. Can we parameterize it here, and pass in when used?